code/game/objects/items/storage/belt.dm

```diff
@@ -12,6 +12,7 @@
 	attack_verb_simple = list("whip", "lash", "discipline")
 	max_integrity = 300
 	equip_sound = 'sound/items/equip/toolbelt_equip.ogg'
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
goofball and single-line webedits, name a more iconic duo

Reply:
athath and untested PRs